ragen/env/deepcoder/env.py

```
@@ -0,0 +1,190 @@
+from typing import Any, Dict, Optional, Tuple
+
+from datasets import load_dataset
+from ragen.env.base import BaseLanguageBasedEnv
+from ragen.utils import all_seed
+from .config import DeepCoderEnvConfig
+from .utils import prepare_deepcoder_data, run_deepcoder_sandbox
+
+
+class DeepCoderEnv(BaseLanguageBasedEnv):
+    def __init__(self, config: Optional[DeepCoderEnvConfig] = None):
+        super(DeepCoderEnv, self).__init__()
```

Ruff (0.15.2)

[warning] Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)